udata/core/spatial/models.py

```diff
@@ -101,11 +104,17 @@ def keys_values(self):
         return keys_values
 
     @cached_property
+    def level_code(self):
+        """Truncated level code for the sake of readability."""
+        # Either 'region', 'departement' or 'commune'.
+        return self.id.split(':')[1]
```

Comment:
It's linked to `TERRITORY_DATASETS` which doesn't contain countries. It's not in use for display but to load datasets defined in udata-gouvfr